.github/workflows/cypress-tests.yml

```diff
@@ -0,0 +1,51 @@
+name: E2E Tests
+
+on:
+  push:
+    branches: [patcon/236-cypress-gha]
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-16.04
+    env:
+      # Use native docker command within docker-compose
+      COMPOSE_DOCKER_CLI_BUILD: 1
+      # Use buildkit to speed up docker command
+      DOCKER_BUILDKIT: 1
+    steps:
+      - name: Checkout app repo
+        uses: actions/checkout@v2
+
+      - name: Build full project via docker-compose
+        run: docker-compose build --parallel
+
+      - run: docker-compose up --detach
+
+      - run: |
+          echo '{"projectId": "4rbrm6"}' > e2e/cypress.json
+
+      - uses: cypress-io/github-action@v1
+        env:
+          CYPRESS_BASE_URL: http://localhost
+          CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
```

Comment:
I think we can leave this unset. Just means pretty results don't get pushed to the free Cypress.io dashboard.

Also, happy to share my per-project key that I used, which we can use to avoid getting bogged down with creating and org account on the hosted cypress.io dashboard.

(this dashboard is totally optional btw)